data/coalition.txt

```diff
@@ -919,4 +920,4 @@ phrase "hostile heliarch"
 		"not die"
 		"not be extinct"
 	word
-		"!"
```

Comment:
Unrelated change.

Reply:
That's weird, I didn't touch the last line; I have no idea why github makes it show up.

Reply:
The file in `master` has no newline at the end of the file. Nescio's version has one. Moving forward, those who use editors that support EditorConfig rules will ensure that the EOF newline is present.